.yarnrc.yml

```diff
@@ -324,26 +308,33 @@ packageExtensions:
         optional: true
   '@automattic/isolated-block-editor@2.4.1':
     peerDependencies:
-      # '@wp-g2/create-styles': '*'
       react: '*'
       react-dom: '*'
       redux: '*'
-      react-native: '*'
   jest-enzyme@7.1.2:
     peerDependencies:
       react: '*'
   '@automattic/newspack-blocks@*':
     peerDependencies:
       react: '*'
       react-dom: '*'
-      react-native: '*'
       reakit-utils: '*'
   newspack-components@*:
+    dependencies:
+      react-native: '*' # TODO: Remove when newspack-components updates.
     peerDependencies:
       '@wordpress/data': '*'
       react: '*'
       react-dom: '*'
-      react-native: '*'
+  react-native-codegen@*: # TODO: Remove when newspack-components updates.
```

Comment:
Should we pin the version number here? I think `yarn` will let us know when the rule doesn't apply anymore if the version is pinned